etc/ui-config/mh_default_org/theodul/config.yml

```diff
@@ -0,0 +1,168 @@
+# Default: false
+show_embed_links: true
+
+# Shortcut definitions for the html5 player
+# - modifier keys
+#   - shift
+#   - ctrl
+#   - alt
+#   - option
+#   - meta
+#   - command
+#   - mod -- combines Mac 'command' and Windows/Linux 'ctrl'
+# - special keys
+#   - backspace
+#   - tab
+#   - enter
+#   - return
+#   - capslock
+#   - esc
+#   - escape
+#   - space
+#   - pageup
+#   - pagedown
+#   - end
+#   - home
+#   - left
+#   - up
+#   - right
+#   - down
+#   - ins
+#   - del
+#   - plus
+# - any other key
+#   - should be referenceable by name like a, /, $, *, or =
+# - for more information have a look at the player docs
+shortcuts:
+  controls:
+    playPause: space
+    seekRight: right
+    seekLeft: left
+    jumpToBegin: backspace
+    prevChapter: pagedown
+    nextChapter: pageup
+  playbackrate:
+    increase: t
+    decrease: T
+  volume:
+    up: 9
+    down: 8
+  fullscreen:
+    enable: mod+enter
+    cancel: escape
+  layout:
+    focusNext: '>'
+    focusPrev: '<'
+    movePip: p
+    togglePiP: o
+  zoom:
+    moveLeft: a
+    moveRight: d
+    moveUp: w
+    moveDown: s
+    in: plus
+    out: '-'
+
+
+################################################################################
+# Matomo (User Tracking) Configuration
+#
+# These settings allow a tracking of user interaction with Matomo (https://matomo.org/) in the Opencast Player
+# The plugin will be activated only if a Matomo server.
+################################################################################
+matomo:
+
+  # The Matomo server from which the Piwik JS library will be loaded and to which the data will be reported
+  #server: http://localhost/matomo
+
+  # The Matomo site ID has to be numeric value. If not set this will be 1.
+  #site_id:1
+
+  # The heartbeat setting to track how long a user stayed on the player page. Set to 0 or comment this line to
+  # disable the heartbeat
+  #heartbeat:30
+
+  # Notification that matomo is tracking the user, set to false to disable notifications
+  # Default: true
+  #notification:true
+
+  # These settings let you track several player events. Add the events that you want to track to the list.
+  # Events that can be tracked:
+  # - play: play has been pressed (will also be called if after seeking).
+  # - pause: pause has been pressend (will also be called if before seeking).
+  # - seek: user jumps to a different time. Time in seconds will be stored
+  # - ended: video has reached the end
+  # - playbackrate: user changes the playback speed (values 0.75 to 3.00)
+  # - volume: Volume change by the user value 0.0 to 1.0
+  # - quality: manual change of video quality (quality tag is stored)
+  # - fullscreen: user presses fullscreen button
+  # - focus: user selects one video to be enlarged (flavor of selected video is stored)
+  # - layout_reset: user switches back to default layout
+  # - zoom: user changes the zoom of the video
+  #track_events:
+  #  - play
+  #  - ended
```

Comment:
Maybe something like
```yaml
  # These settings let you track several player events. Add the events that you want to track to the list.
  # Events that can be tracked:
  #
  #track_events:
  #  - play         # play has been pressed (will also be called if after seeking).
  #  - pause        # pause has been pressend (will also be called if before seeking).
  #  - seek         # user jumps to a different time. Time in seconds will be stored
  #  - ended        # video has reached the end
  #  - playbackrate # user changes the playback speed (values 0.75 to 3.00)
  #  - volume       # Volume change by the user value 0.0 to 1.0
  #  - quality      # manual change of video quality (quality tag is stored)
  #  - fullscreen   # user presses fullscreen button
  #  - focus        # user selects one video to be enlarged (flavor of selected video is stored)
  #  - layout_reset # user switches back to default layout
  #  - zoom         # user changes the zoom of the video
```
would be better? In the old configuration you could just uncomment and track everything by default. Also, this prevents that the list is repeated twice.

Reply:
Great suggestion! It looks cleaner and easy to read. I will modify the file.